presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestMinMaxByAggregation.java

```diff
@@ -84,7 +82,8 @@ public void testAllRegistered()
 
     private static List<Type> getTypes()
     {
-        List<Type> simpleTypes = METADATA.getTypeManager().getTypes();
+        List<Type> simpleTypes = new ArrayList<>(METADATA.getTypeManager().getTypes());
+        simpleTypes.removeAll(ImmutableList.of(CODE_POINTS, JSON_PATH, JONI_REGEXP, RE2J_REGEXP, LIKE_PATTERN));
```

Comment:
StateCompiler doesn't work for these types. @haozhun said they are internal types so we don't need to support them. If needed I can look into it further.